app/views/internal/privileged_reactions/index.html.erb

```diff
@@ -2,14 +2,14 @@
   <div class="col">
   </div>
   <div class="col">
-    <%= search_form_for @q, url: internal_negative_reactions_path, class: "form-inline justify-content-end" do |f| %>
+    <%= search_form_for @q, url: internal_privileged_reactions_path, class: "form-inline justify-content-end" do |f| %>
 
       <%= f.label :user_username_cont, "User", class: "sr-only" %>
       <%= f.search_field :user_username_cont, placeholder: "User", class: "form-control mx-3" %>
 
       <%= f.select(
             :category_eq,
-            options_for_select(["", "vomit", "thumbsdown"], @q.category_eq),
+            options_for_select(["", "thumbsup", "thumbsdown", "vomit"], @q.category_eq),
```

Comment:
another place where we could use the constant "moderator reactions" or "privileged reactions"